src/main/service/subscriptionService.ts

```diff
@@ -66,4 +73,119 @@ export class SubscriptionService {
     }
   }
 
+  public async handleNewSubscription(pendingSubscription, user): Promise<void> {
+    const selectionList = Object.keys(pendingSubscription);
+    for (const selectionName of selectionList) {
+      let hearingIdsList = [];
+      let courtIdsList = [];
+      let caseDetailsList: Hearing[] = [];
+      let courtDetailsList: Court[] = [];
+      switch (selectionName) {
+        case 'case-number':
+        case 'hearing-selections[]':
+          // pendingSubscription.selectionName gives undefined
```

Comment:
can yeet this comment out

Reply:
might be worth keeping as it is a reference to approach taken on the line bellow